src/main/java/com/dnastack/ga4gh/dataconnect/adapter/trino/TrinoHttpClient.java

```diff
@@ -248,7 +247,9 @@ private Response post(String url, String body, Map<String, String> extraCredenti
     private Response execute(final Request.Builder request, Map<String, String> extraCredentials) throws IOException {
         request.header("X-Trino-User", getUserNameForRequest());
         request.header("X-Trino-Trace-Token",tracing.currentTraceContext().get().traceIdString());
-        request.header("X-Trino-Extra-Credential", "trace=" + tracing.currentTraceContext().get().traceIdString()); // We're doing this to extract the trace token inside the SAC plugin
+        String b3Header = String.format("%s-%s-%s-%s", tracing.currentTraceContext().get().traceIdString(), tracing.currentTraceContext().get().spanIdString(),
```

Comment:
I can see the old code does a `tracing.currentTraceContext().get().traceIdString()` so apparently this isn't an issue... but it does say in the Brave javadoc that CurrentTraceContext.get() will return null if there is no current context. Maybe best to put the code that requires a current context into an if block?